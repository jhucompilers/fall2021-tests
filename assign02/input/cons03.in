// function to add n to each member of the list lst,
// returning a list
function addtomembers(lst, n) {
  var result;
  if (nilp(lst)) {
    result = lst;
  } else {
    result = cons(car(lst) + n, addtomembers(cdr(lst), n));
  }
  result;
}

// result should be (11 12 13)
addtomembers(list(1, 2, 3), 10);
