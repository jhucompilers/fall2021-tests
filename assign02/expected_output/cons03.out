Result: (11 12 13)
